lib/src/utils/ne_utils.dart

```diff
@@ -0,0 +1,26 @@
+class NepaliLangUtils {
+  static getNepaliNumber(int num) {
+    final List<String> nepaliDigits = [
+      '०',
+      '१',
+      '२',
+      '३',
+      '४',
+      '५',
+      '६',
+      '७',
+      '८',
+      '९'
+    ];
+
+    String result = '';
+    String numberString = num.toString();
+
+    for (int i = 0; i < numberString.length; i++) {
+      int digit = int.parse(numberString[i]);
+      result += nepaliDigits[digit];
```

Comment:
Here we can use `StringBuffer()` instead of string concatenation to be efficient with memory